integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/cloud/v2/provider/AwsCloudProvider.java

```diff
@@ -50,11 +50,11 @@ private AwsNetworkV4Parameters networkParameters() {
     }
 
     public String getVpcId() {
-        return getTestParameter().getWithDefault(AwsParameters.VPC_ID, "vpc-578e5f31");
+        return getTestParameter().getWithDefault(AwsParameters.VPC_ID, "vpc-5e68eb3a");
```

Comment:
As we agreed you shall create separate classes for the default values.

Reply:
It was just an advice for you how to do...